stencil-library/src/components/payments-list/payments-list.tsx

```diff
@@ -59,22 +62,22 @@ export class PaymentsList {
   }
 
   emptyState = (
-    <tr part="empty-state">
-      <td colSpan={8} style={{ textAlign: 'center' }}>No payments to show</td>
+    <tr>
+      <td part="empty-state" colSpan={8} style={{ textAlign: 'center' }}>No payments to show</td>
     </tr>
   );
 
   errorState = () => (
-    <tr part="error-state">
-      <td colSpan={8} style={{ textAlign: 'center' }}>
-        <span>An unexpected error occurred: {this.errorMessage}</span>
+    <tr>
+      <td part="error-state" colSpan={8} style={{ textAlign: 'center' }}>
+        An unexpected error occurred: {this.errorMessage}
       </td>
     </tr>
   );
 
   loadingState = (
-    <tr part="loading-state">
-      <td colSpan={8} style={{ textAlign: 'center' }}>
+    <tr>
+      <td part="loading-state" colSpan={8} style={{ textAlign: 'center' }}>
         <div class="spinner-border" role="status">
```

Comment:
We may want `loading-state-spinner` as well - I can definitely imagine that some folks will want to change the color of the spinner, as well as other things like size.